sites/dedeman.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import json
+from utils import *
+
+url = 'https://recrutare.dedeman.ro/sinapsi-jobannounceswithfilter/'
+company = 'DEDEMAN'
+
+data = {
+    'request': {
+        'JobAnnounces': [
+            {
+                'Id': '',
+                'Function': '',
+                'City': '',
+            }
+        ]
+    }
+}
+
+response_data = requests.post(url, json=data).json()
+job_announces = response_data['d']['JobAnnounces']
+
+final_jobs = []
+
+for job in job_announces:
+    final_jobs.append(
+            create_job(
+                job_title = job['Function'],
+                company = company,
+                city = job['City'],
+                country = 'Romania',
+                job_link = 'https://recrutare.dedeman.ro/detalii-post.php?id=' + job['Id']
+            )
+        )
+
+for version in [1, 4]:
+    publish(version, company, final_jobs, 'Grasum_Key')
+
+publish_logo(company, 'https://recrutare.dedeman.ro/assets/images/logo.svg?v=12')
```

Comment:
Replace the logo url with [https://i.dedeman.ro/dedereact/design/images/logo.svg](url)
```suggestion
publish_logo(company, 'https://i.dedeman.ro/dedereact/design/images/logo.svg')
```
It looks like this 
<img width="194" alt="Screenshot 2023-08-11 at 22 08 46" src="https://github.com/peviitor-ro/based_scraper_py/assets/67306273/f488cf00-2b1e-49ea-82db-395300babec5">
not like this
<img width="1440" alt="Screenshot 2023-08-11 at 22 13 21" src="https://github.com/peviitor-ro/based_scraper_py/assets/67306273/2135d660-d6c0-49de-b33f-748bbdd0b5a9">

Reply:
Logo was taken from their recruiting page.